Formula/tippecanoe.rb

```diff
@@ -1,8 +1,8 @@
 class Tippecanoe < Formula
   desc "Build vector tilesets from collections of GeoJSON features"
   homepage "https://github.com/mapbox/tippecanoe"
-  url "https://github.com/mapbox/tippecanoe/archive/1.9.13.tar.gz"
-  sha256 "65d4b63301e8b6c02577753cc177de74630aaba4d53dd669f4e5da67b4ea1c5f"
+  url "https://github.com/mapbox/tippecanoe/archive/1.9.15.tar.gz"
+  sha256 "e58cac588842a5f70d263d0e9b2eb940e32b06ab3c6560bbd650059484bab184"
 
   bottle do
     cellar :any
```

Comment:
You keep leaving comments that don't make a lot of sense tonight. I'd love to be able to help but can you clarify what you're asking about?